yarn-project/acir-simulator/src/client/simulator.ts

```diff
@@ -166,7 +165,7 @@ export class AcirSimulator {
         abi,
         AztecAddress.ZERO,
         EthAddress.ZERO,
-        PrivateHistoricTreeRoots.empty(),
+        ConstantHistoricBlockData.empty(),
```

Comment:
Why is this a better name that something like `HistoricBlockDataTreeRoots` (like its predecessor)?

Reply:
It's no longer just roots, as the global variables are included within it

Reply:
Hmm, I suppose this can be thought of as a block header cant it?